src/components/form/examples/row-layout-schema.ts

```diff
@@ -0,0 +1,107 @@
+export const schema = {
+    description: 'This form has the row layout',
+    type: 'object',
+    properties: {
+        info: {
+            title: 'Preferences',
+            description:
+                'These settings will not affect how others see the data. These are only for you.',
+            type: 'object',
+            lime: {
+                layout: {
+                    type: 'row',
+                },
+            },
+            properties: {
+                language: {
+                    type: 'string',
+                    icon: 'globe',
+                    title: 'Language',
+                    description: 'Select the app language',
+                    default: 'ua',
+                    lime: {
+                        component: 'limel-select',
```

Comment:
This is incorrect. It should be
```suggestion
                        component: { name: 'limel-select' },
```